libs/blocks/merch-twp/merch-twp.css

```diff
@@ -0,0 +1,21 @@
+@media (max-width: 1199px) {
```

Comment:
We should have a mobile first approach and build our way to desktop.

Reply:
This rule forces the milo modal to take the whole screen on mobile/tablet devices (requirement for TWP).
The mobile first responsive styles are packaged with the web-component in `deps/merch-twp-d2p.js`.

Reply:
Also, for TWP,  layout mobile is different than desktop, we render different markups, so not relaying on CSS only for TWP responsiveness.